recipes/verible/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "verible" %}
+{% set version = "0.0-3667-g88d12889" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/chipsalliance/verible/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6a13a902bfd37ecabfd772d619251da40e8ad8e44cf75ec2bc8663046200b02a
+  patches:
+    - build_opts_patch.patch
+
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - bazel >=5,<=7
+    - bazel-toolchain
+    - flex
+    - bison
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - patchelf
+  host:
+    - python
+  run:
+    - python
```

Comment:
Why do you need python here?

Reply:
Yep - was able to rerun local without any python requirement and it went through all tests successfully - the build section in the source repo suggested python as required to build, so I hadn't tried yet without. Good catch @isuruf